src/plugins/orangehrmClaimPlugin/Dto/ClaimRequestSearchFilterParams.php

```diff
@@ -144,6 +144,9 @@ public function getToDate(): ?DateTime
      */
     public function setToDate(?DateTime $toDate): void
     {
+        if ($toDate !== null) {
```

Comment:
```suggestion
        if ($toDate instanceof DateTime) {
```
I prefer this. but this is fine as param type already defined